docs/new-guides.md

```diff
@@ -0,0 +1,23 @@
+# Proposing a new guide
+
+Thanks for your interest in suggesting a new Open Source Guide! Your suggestions help us decide what to work on next.
+
+## If you'd like to propose a new guide:
+
+1. **[Open an issue](https://github.com/github/opensource.guide/issues)** with your suggestion. Please explain:
+  * Why the topic is broadly useful to open source contributors
+  * Why it doesn't fit into an existing guide
+  * 3-5 bullets of expected sub-topics
+2. We (likely @bkeepers or @nayafia) will label the issue ```new-guide``` and share feedback.
+
+At this point, we may decide the topic doesn't warrant a new guide and close the issue, or we may recommend adding the information to an existing guide. That's okay! All suggestions are welcomed contributions, because they help us understand what our community is thinking.
+
+## If we decide to move forward on a new guide:
+
+1. We'll add a ```research``` label to your issue.
+2. We'll start using your issue to collect existing community resources (links, blog posts, projects). Everybody is welcome to contribute.
+3. We'll open a PR with an outline for the new guide. Anyone is welcome to give feedback.
+4. After general consensus has been reached, your issue will be marked ```in progress```. We'll add the guide to our editorial schedule and estimate a date for 1st draft completion. We'll also put in a request with Design for illustrations.
+5. The PR will be updated with the 1st draft. Your issue will be marked ```1st draft```. Anyone is welcome to give feedback.
```

Comment:
Maybe just `draft` label, since it'll likely go through a few iterations of editing.